internal/nodes/dequeue_ingestion.go

```diff
@@ -194,6 +199,11 @@ func handlePod(key, namespace, podName string, fullsync bool) {
 			if ok {
 				utils.AviLog.Debugf("key: %s, msg: handling l4 Services %v", key, lbSvcs)
 				for _, lbSvc := range lbSvcs {
+					ns := strings.Split(lbSvc, "/")
+					//Do not append L4 service if namespace is invalid
+					if !utils.IsServiceNSValid(ns[0]) {
```

Comment:
I think we can do this check only once, as all the services would be in the same NS